pkg/components/rook/component.go

```diff
@@ -39,6 +41,59 @@ type component struct {
 	DiscoverTolerationEffect string `hcl:"discover_toleration_effect,optional"`
 }
 
+const chartValuesTmpl = `
+{{- if .NodeSelectors }}
+nodeSelector:
+  {{- range $key, $value := .NodeSelectors }}
+  {{ $key }}: "{{ $value }}"
+  {{- end }}
+{{- end }}
+
+{{- if .Tolerations }}
+tolerations: {{ .TolerationsRaw }}
+{{- end }}
+
+csi:
+  cephfsGrpcMetricsPort: 9092
```

Comment:
Can we have a comment here, that without this it clashes with Calico metrics port?